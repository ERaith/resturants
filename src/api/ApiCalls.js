export async function fetchData() {
  const res = await fetch(
    "https://code-challenge.spectrumtoolbox.com/api/restaurants",
    {
      method: "GET",
      headers: {
        "Content-Type": "application/json",
        Authorization: "Api-Key q3MNxtfep8Gt",
      },
    }
  );
  const data = await res.json();
  console.log(data);
  return alphebetize(data);
}

function alphebetize(data) {
  return data.sort((a, b) => {
    if (a.name < b.name) {
      return -1;
    }
    if (a.name > b.name) {
      return 1;
    }
    return 0;
  });
}
