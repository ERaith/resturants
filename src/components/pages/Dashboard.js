import React, { useEffect, useState } from "react";

import { Paginator } from "../paginator/Paginator";
import { SearchBar } from "../searchBar/SearchBar";
import { SelectionFilter } from "../selectionFilter/SelectionFilter";
import { Table } from "../table/Table";
import { states } from "../../api/data";

const pageSize = 10;

function Dashboard({ data, headerMeta,genreFilterKeys }) {
  const [genreFilter, setGenreFilter] = useState("");
  const [genreFilterActive, setGenreFilterActive] = useState(true);
  const [searchTerm, setSearchTerm] = useState("");
  const [stateFilter, setStateFilter] = useState("");
  const [stateFilterActive, setStateFilterActive] = useState(true);
  const [tableData, setTableData] = useState([]);

  const handleSearchChange = (value) => {
    setSearchTerm(value);
  };

  const handleChange = (event) => {
    switch (event.target.id) {
      case "state selection":
        setStateFilter(event.target.value);
        break;
      case "state selection:isActive":
        setStateFilterActive(event.target.checked);
        break;
      case "genre selection":
        setGenreFilter(event.target.value);
        break;

      case "genre selection:isActive":
        setGenreFilterActive(event.target.checked);
        break;
      default:
        break;
    }
  };

  useEffect(() => {
    setTableData(data);
  }, [data]);

  useEffect(() => {
    const results = data.filter((resturant) => {
      const name = isPresent(resturant, "name", searchTerm);
      const city = isPresent(resturant, "city", searchTerm);
      const genre = isPresent(resturant, "genre", searchTerm);
      const search = name || city || genre;

      const statePresent =
        stateFilterActive &&
        stateFilter !== "All" &&
        !resturant.state.includes(stateFilter);

      const genrePresent =
        genreFilterActive &&
        genreFilter !== "All" &&
        !resturant.genre.includes(genreFilter);

      return search && !statePresent && !genrePresent;
    });

    setTableData(results);
  }, [
    data,
    genreFilter,
    genreFilterActive,
    searchTerm,
    stateFilter,
    stateFilterActive,
  ]);

  return (
    <div className="main">
      <div className="search-container">
        <SearchBar handleSearchChange={handleSearchChange} />
        <SelectionFilter
          dropDownItems={states}
          filterTermActive={stateFilterActive}
          handleChange={handleChange}
          id="state selection"
        />
        <SelectionFilter
          dropDownItems={genreFilterKeys}
          filterTermActive={genreFilterActive}
          handleChange={handleChange}
          id="genre selection"
        />
      </div>
      <div className="table-container">
        <Table tableData={tableData} headerMeta={headerMeta} />
      </div>
      <Paginator
        size={Math.ceil(data.length / pageSize)}
      />
    </div>
  );
}

export default Dashboard;

export function isPresent(data, key, searchTerm) {
  return data[key].toLowerCase().includes(searchTerm.toLowerCase());
}
