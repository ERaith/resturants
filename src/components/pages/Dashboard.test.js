import React from "react";
import toJson from "enzyme-to-json";

import Dashboard from "./Dashboard";

import { headerMeta } from "../../api/data";
import { mount } from "enzyme";
import { resturants } from "../../api/data";

describe("Dashboard", () => {
  describe("Snapshots", () => {
    it("should match the initial snapshot", () => {
      const props = {
        headerMeta: headerMeta,
        data: resturants.slice(0, 4),
        genreFilterKeys: ["American", "Seafood"],
      };
      const wrapper = mount(<Dashboard {...props} />);

      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
