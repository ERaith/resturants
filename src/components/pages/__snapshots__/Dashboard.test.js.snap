// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Snapshots should match the initial snapshot 1`] = `
<Dashboard
  data={
    Array [
      Object {
        "address1": "201 Waterfront St",
        "attire": "business casual",
        "city": "Oxon Hill",
        "genre": "Steak,American,Contemporary,Seafood,Cafe",
        "hours": "Open Daily 5:30 PM-10:00 PM",
        "id": "f223fdd0-4adc-423e-9747-980a66c256ca",
        "lat": "38.782098",
        "long": "-77.017492",
        "name": "Old Hickory Steakhouse",
        "state": "MD",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses",
        "telephone": "(301) 965-4000",
        "website": "http://www.gaylordnational.com",
        "zip": "20745",
      },
      Object {
        "address1": "303 E Main St",
        "attire": "business casual",
        "city": "Aspen",
        "genre": "Japanese,Sushi,Asian,Contemporary,Seafood",
        "hours": "Open Daily 5:30 PM-9:00 PM",
        "id": "00b35e1a-82b1-4988-b8b9-6df826db2818",
        "lat": "39.190723",
        "long": "-106.82031",
        "name": "Matsuhisa",
        "state": "CO",
        "tags": "Social,Food and Dining,Restaurants,Japanese,Social,Food and Dining,Restaurants,Sushi",
        "telephone": "(970) 544-6628",
        "website": "http://www.matsuhisaaspen.com",
        "zip": "81611",
      },
      Object {
        "address1": "108 3rd St NE",
        "attire": "business casual",
        "city": "Charlottesville",
        "genre": "French,European,Cafe,Continental,American",
        "hours": "Mon-Thu 5:30 PM-9:00 PM; Fri-Sat 5:30 PM-10:00 PM",
        "id": "0f41a3d0-0641-4eef-b7fd-706f083cf0d5",
        "lat": "38.030526",
        "long": "-78.479785",
        "name": "Fleurie Restaurant",
        "state": "VA",
        "tags": "Social,Food and Dining,Restaurants,French",
        "telephone": "(434) 971-7800",
        "website": "http://www.fleurierestaurant.com",
        "zip": "22902",
      },
      Object {
        "address1": "500 Crescent Ct",
        "attire": "business casual",
        "city": "Dallas",
        "genre": "Steak,American",
        "hours": "Mon-Thu 11:00 AM-10:00 PM; Fri 11:00 AM-11:00 PM; Sat 5:00 PM-11:00 PM; Sun 5:00 PM-9:00 PM",
        "id": "0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56",
        "lat": "32.794749",
        "long": "-96.804099",
        "name": "The Capital Grille",
        "state": "TX",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses,Social,Food and Dining,Restaurants,American",
        "telephone": "(214) 303-0500",
        "website": "http://www.thecapitalgrille.com",
        "zip": "75201",
      },
    ]
  }
  genreFilterKeys={
    Array [
      "American",
      "Seafood",
    ]
  }
  headerMeta={
    Array [
      Object {
        "key": "name",
        "text": "Resturant Name",
      },
      Object {
        "key": "city",
        "text": "CITY",
      },
      Object {
        "key": "state",
        "text": "STATE",
      },
      Object {
        "key": "telephone",
        "text": "TELEPHONE",
      },
      Object {
        "key": "genre",
        "text": "GENRE",
      },
    ]
  }
>
  <div
    className="main"
  >
    <div
      className="search-container"
    >
      <SearchBar
        handleSearchChange={[Function]}
      >
        <form
          onSubmit={[Function]}
        >
          <label
            htmlFor="search"
          >
             SEARCH 
          </label>
          <input
            className="search"
            id="search"
            onChange={[Function]}
            placeholder="Search"
            type="text"
            value=""
          />
          <button
            className="submit-search"
            type="submit"
            value="Submit"
          >
            Submit
          </button>
          <button
            className="clear-search"
            onClick={[Function]}
            type="reset"
          >
            Clear Search
          </button>
        </form>
      </SearchBar>
      <SelectionFilter
        dropDownItems={
          Array [
            "AL",
            "AK",
            "AZ",
            "AR",
            "CA",
            "CO",
            "CT",
            "DE",
            "FL",
            "GA",
            "HI",
            "ID",
            "IL",
            "IN",
            "IA",
            "KS",
            "KY",
            "LA",
            "ME",
            "MD",
            "MA",
            "MI",
            "MN",
            "MS",
            "MO",
            "MT",
            "NE",
            "NV",
            "NH",
            "NJ",
            "NM",
            "NY",
            "NC",
            "ND",
            "OH",
            "OK",
            "OR",
            "PA",
            "RI",
            "SC",
            "SD",
            "TN",
            "TX",
            "UT",
            "VT",
            "VA",
            "WA",
            "WV",
            "WI",
            "WY",
          ]
        }
        filterTermActive={true}
        handleChange={[Function]}
        id="state selection"
      >
        <div
          className="filter-container"
        >
          <label
            htmlFor="state selection"
          >
             
            STATE SELECTION
             
          </label>
          <select
            className="filter"
            id="state selection"
            key="state selection"
            onChange={[Function]}
          >
            <option
              defaultValue={true}
              key="Default-Option:state selection"
            >
              All
            </option>
            <option
              key="AL"
            >
              AL
            </option>
            <option
              key="AK"
            >
              AK
            </option>
            <option
              key="AZ"
            >
              AZ
            </option>
            <option
              key="AR"
            >
              AR
            </option>
            <option
              key="CA"
            >
              CA
            </option>
            <option
              key="CO"
            >
              CO
            </option>
            <option
              key="CT"
            >
              CT
            </option>
            <option
              key="DE"
            >
              DE
            </option>
            <option
              key="FL"
            >
              FL
            </option>
            <option
              key="GA"
            >
              GA
            </option>
            <option
              key="HI"
            >
              HI
            </option>
            <option
              key="ID"
            >
              ID
            </option>
            <option
              key="IL"
            >
              IL
            </option>
            <option
              key="IN"
            >
              IN
            </option>
            <option
              key="IA"
            >
              IA
            </option>
            <option
              key="KS"
            >
              KS
            </option>
            <option
              key="KY"
            >
              KY
            </option>
            <option
              key="LA"
            >
              LA
            </option>
            <option
              key="ME"
            >
              ME
            </option>
            <option
              key="MD"
            >
              MD
            </option>
            <option
              key="MA"
            >
              MA
            </option>
            <option
              key="MI"
            >
              MI
            </option>
            <option
              key="MN"
            >
              MN
            </option>
            <option
              key="MS"
            >
              MS
            </option>
            <option
              key="MO"
            >
              MO
            </option>
            <option
              key="MT"
            >
              MT
            </option>
            <option
              key="NE"
            >
              NE
            </option>
            <option
              key="NV"
            >
              NV
            </option>
            <option
              key="NH"
            >
              NH
            </option>
            <option
              key="NJ"
            >
              NJ
            </option>
            <option
              key="NM"
            >
              NM
            </option>
            <option
              key="NY"
            >
              NY
            </option>
            <option
              key="NC"
            >
              NC
            </option>
            <option
              key="ND"
            >
              ND
            </option>
            <option
              key="OH"
            >
              OH
            </option>
            <option
              key="OK"
            >
              OK
            </option>
            <option
              key="OR"
            >
              OR
            </option>
            <option
              key="PA"
            >
              PA
            </option>
            <option
              key="RI"
            >
              RI
            </option>
            <option
              key="SC"
            >
              SC
            </option>
            <option
              key="SD"
            >
              SD
            </option>
            <option
              key="TN"
            >
              TN
            </option>
            <option
              key="TX"
            >
              TX
            </option>
            <option
              key="UT"
            >
              UT
            </option>
            <option
              key="VT"
            >
              VT
            </option>
            <option
              key="VA"
            >
              VA
            </option>
            <option
              key="WA"
            >
              WA
            </option>
            <option
              key="WV"
            >
              WV
            </option>
            <option
              key="WI"
            >
              WI
            </option>
            <option
              key="WY"
            >
              WY
            </option>
          </select>
          <input
            checked={true}
            className="checkbox"
            id="state selection:isActive"
            onChange={[Function]}
            type="checkbox"
          />
        </div>
      </SelectionFilter>
      <SelectionFilter
        dropDownItems={
          Array [
            "American",
            "Seafood",
          ]
        }
        filterTermActive={true}
        handleChange={[Function]}
        id="genre selection"
      >
        <div
          className="filter-container"
        >
          <label
            htmlFor="genre selection"
          >
             
            GENRE SELECTION
             
          </label>
          <select
            className="filter"
            id="genre selection"
            key="genre selection"
            onChange={[Function]}
          >
            <option
              defaultValue={true}
              key="Default-Option:genre selection"
            >
              All
            </option>
            <option
              key="American"
            >
              American
            </option>
            <option
              key="Seafood"
            >
              Seafood
            </option>
          </select>
          <input
            checked={true}
            className="checkbox"
            id="genre selection:isActive"
            onChange={[Function]}
            type="checkbox"
          />
        </div>
      </SelectionFilter>
    </div>
    <div
      className="table-container"
    >
      <Table
        headerMeta={
          Array [
            Object {
              "key": "name",
              "text": "Resturant Name",
            },
            Object {
              "key": "city",
              "text": "CITY",
            },
            Object {
              "key": "state",
              "text": "STATE",
            },
            Object {
              "key": "telephone",
              "text": "TELEPHONE",
            },
            Object {
              "key": "genre",
              "text": "GENRE",
            },
          ]
        }
        tableData={
          Array [
            Object {
              "address1": "201 Waterfront St",
              "attire": "business casual",
              "city": "Oxon Hill",
              "genre": "Steak,American,Contemporary,Seafood,Cafe",
              "hours": "Open Daily 5:30 PM-10:00 PM",
              "id": "f223fdd0-4adc-423e-9747-980a66c256ca",
              "lat": "38.782098",
              "long": "-77.017492",
              "name": "Old Hickory Steakhouse",
              "state": "MD",
              "tags": "Social,Food and Dining,Restaurants,Steakhouses",
              "telephone": "(301) 965-4000",
              "website": "http://www.gaylordnational.com",
              "zip": "20745",
            },
            Object {
              "address1": "303 E Main St",
              "attire": "business casual",
              "city": "Aspen",
              "genre": "Japanese,Sushi,Asian,Contemporary,Seafood",
              "hours": "Open Daily 5:30 PM-9:00 PM",
              "id": "00b35e1a-82b1-4988-b8b9-6df826db2818",
              "lat": "39.190723",
              "long": "-106.82031",
              "name": "Matsuhisa",
              "state": "CO",
              "tags": "Social,Food and Dining,Restaurants,Japanese,Social,Food and Dining,Restaurants,Sushi",
              "telephone": "(970) 544-6628",
              "website": "http://www.matsuhisaaspen.com",
              "zip": "81611",
            },
            Object {
              "address1": "108 3rd St NE",
              "attire": "business casual",
              "city": "Charlottesville",
              "genre": "French,European,Cafe,Continental,American",
              "hours": "Mon-Thu 5:30 PM-9:00 PM; Fri-Sat 5:30 PM-10:00 PM",
              "id": "0f41a3d0-0641-4eef-b7fd-706f083cf0d5",
              "lat": "38.030526",
              "long": "-78.479785",
              "name": "Fleurie Restaurant",
              "state": "VA",
              "tags": "Social,Food and Dining,Restaurants,French",
              "telephone": "(434) 971-7800",
              "website": "http://www.fleurierestaurant.com",
              "zip": "22902",
            },
            Object {
              "address1": "500 Crescent Ct",
              "attire": "business casual",
              "city": "Dallas",
              "genre": "Steak,American",
              "hours": "Mon-Thu 11:00 AM-10:00 PM; Fri 11:00 AM-11:00 PM; Sat 5:00 PM-11:00 PM; Sun 5:00 PM-9:00 PM",
              "id": "0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56",
              "lat": "32.794749",
              "long": "-96.804099",
              "name": "The Capital Grille",
              "state": "TX",
              "tags": "Social,Food and Dining,Restaurants,Steakhouses,Social,Food and Dining,Restaurants,American",
              "telephone": "(214) 303-0500",
              "website": "http://www.thecapitalgrille.com",
              "zip": "75201",
            },
          ]
        }
      >
        <table
          className="container"
          id="resturants"
        >
          <tbody>
            <TableHeader
              headers={
                Array [
                  Object {
                    "key": "name",
                    "text": "Resturant Name",
                  },
                  Object {
                    "key": "city",
                    "text": "CITY",
                  },
                  Object {
                    "key": "state",
                    "text": "STATE",
                  },
                  Object {
                    "key": "telephone",
                    "text": "TELEPHONE",
                  },
                  Object {
                    "key": "genre",
                    "text": "GENRE",
                  },
                ]
              }
            >
              <tr
                className="table-row"
              >
                <th
                  key="name"
                >
                  Resturant Name
                </th>
                <th
                  key="city"
                >
                  CITY
                </th>
                <th
                  key="state"
                >
                  STATE
                </th>
                <th
                  key="telephone"
                >
                  TELEPHONE
                </th>
                <th
                  key="genre"
                >
                  GENRE
                </th>
              </tr>
            </TableHeader>
            <TableBody
              data={
                Array [
                  Object {
                    "address1": "201 Waterfront St",
                    "attire": "business casual",
                    "city": "Oxon Hill",
                    "genre": "Steak,American,Contemporary,Seafood,Cafe",
                    "hours": "Open Daily 5:30 PM-10:00 PM",
                    "id": "f223fdd0-4adc-423e-9747-980a66c256ca",
                    "lat": "38.782098",
                    "long": "-77.017492",
                    "name": "Old Hickory Steakhouse",
                    "state": "MD",
                    "tags": "Social,Food and Dining,Restaurants,Steakhouses",
                    "telephone": "(301) 965-4000",
                    "website": "http://www.gaylordnational.com",
                    "zip": "20745",
                  },
                  Object {
                    "address1": "303 E Main St",
                    "attire": "business casual",
                    "city": "Aspen",
                    "genre": "Japanese,Sushi,Asian,Contemporary,Seafood",
                    "hours": "Open Daily 5:30 PM-9:00 PM",
                    "id": "00b35e1a-82b1-4988-b8b9-6df826db2818",
                    "lat": "39.190723",
                    "long": "-106.82031",
                    "name": "Matsuhisa",
                    "state": "CO",
                    "tags": "Social,Food and Dining,Restaurants,Japanese,Social,Food and Dining,Restaurants,Sushi",
                    "telephone": "(970) 544-6628",
                    "website": "http://www.matsuhisaaspen.com",
                    "zip": "81611",
                  },
                  Object {
                    "address1": "108 3rd St NE",
                    "attire": "business casual",
                    "city": "Charlottesville",
                    "genre": "French,European,Cafe,Continental,American",
                    "hours": "Mon-Thu 5:30 PM-9:00 PM; Fri-Sat 5:30 PM-10:00 PM",
                    "id": "0f41a3d0-0641-4eef-b7fd-706f083cf0d5",
                    "lat": "38.030526",
                    "long": "-78.479785",
                    "name": "Fleurie Restaurant",
                    "state": "VA",
                    "tags": "Social,Food and Dining,Restaurants,French",
                    "telephone": "(434) 971-7800",
                    "website": "http://www.fleurierestaurant.com",
                    "zip": "22902",
                  },
                  Object {
                    "address1": "500 Crescent Ct",
                    "attire": "business casual",
                    "city": "Dallas",
                    "genre": "Steak,American",
                    "hours": "Mon-Thu 11:00 AM-10:00 PM; Fri 11:00 AM-11:00 PM; Sat 5:00 PM-11:00 PM; Sun 5:00 PM-9:00 PM",
                    "id": "0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56",
                    "lat": "32.794749",
                    "long": "-96.804099",
                    "name": "The Capital Grille",
                    "state": "TX",
                    "tags": "Social,Food and Dining,Restaurants,Steakhouses,Social,Food and Dining,Restaurants,American",
                    "telephone": "(214) 303-0500",
                    "website": "http://www.thecapitalgrille.com",
                    "zip": "75201",
                  },
                ]
              }
              headerOrder={
                Array [
                  "name",
                  "city",
                  "state",
                  "telephone",
                  "genre",
                ]
              }
            >
              <tr
                className="table-row"
                key="f223fdd0-4adc-423e-9747-980a66c256ca"
              >
                <TableCell
                  data="Old Hickory Steakhouse"
                  key="f223fdd0-4adc-423e-9747-980a66c256ca-Old Hickory Steakhouse"
                >
                  <td
                    className="table-cell"
                  >
                    Old Hickory Steakhouse
                  </td>
                </TableCell>
                <TableCell
                  data="Oxon Hill"
                  key="f223fdd0-4adc-423e-9747-980a66c256ca-Oxon Hill"
                >
                  <td
                    className="table-cell"
                  >
                    Oxon Hill
                  </td>
                </TableCell>
                <TableCell
                  data="MD"
                  key="f223fdd0-4adc-423e-9747-980a66c256ca-MD"
                >
                  <td
                    className="table-cell"
                  >
                    MD
                  </td>
                </TableCell>
                <TableCell
                  data="(301) 965-4000"
                  key="f223fdd0-4adc-423e-9747-980a66c256ca-(301) 965-4000"
                >
                  <td
                    className="table-cell"
                  >
                    (301) 965-4000
                  </td>
                </TableCell>
                <TableCell
                  data="Steak,American,Contemporary,Seafood,Cafe"
                  key="f223fdd0-4adc-423e-9747-980a66c256ca-Steak,American,Contemporary,Seafood,Cafe"
                >
                  <td
                    className="table-cell"
                  >
                    Steak,American,Contemporary,Seafood,Cafe
                  </td>
                </TableCell>
              </tr>
              <tr
                className="table-row"
                key="00b35e1a-82b1-4988-b8b9-6df826db2818"
              >
                <TableCell
                  data="Matsuhisa"
                  key="00b35e1a-82b1-4988-b8b9-6df826db2818-Matsuhisa"
                >
                  <td
                    className="table-cell"
                  >
                    Matsuhisa
                  </td>
                </TableCell>
                <TableCell
                  data="Aspen"
                  key="00b35e1a-82b1-4988-b8b9-6df826db2818-Aspen"
                >
                  <td
                    className="table-cell"
                  >
                    Aspen
                  </td>
                </TableCell>
                <TableCell
                  data="CO"
                  key="00b35e1a-82b1-4988-b8b9-6df826db2818-CO"
                >
                  <td
                    className="table-cell"
                  >
                    CO
                  </td>
                </TableCell>
                <TableCell
                  data="(970) 544-6628"
                  key="00b35e1a-82b1-4988-b8b9-6df826db2818-(970) 544-6628"
                >
                  <td
                    className="table-cell"
                  >
                    (970) 544-6628
                  </td>
                </TableCell>
                <TableCell
                  data="Japanese,Sushi,Asian,Contemporary,Seafood"
                  key="00b35e1a-82b1-4988-b8b9-6df826db2818-Japanese,Sushi,Asian,Contemporary,Seafood"
                >
                  <td
                    className="table-cell"
                  >
                    Japanese,Sushi,Asian,Contemporary,Seafood
                  </td>
                </TableCell>
              </tr>
              <tr
                className="table-row"
                key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5"
              >
                <TableCell
                  data="Fleurie Restaurant"
                  key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5-Fleurie Restaurant"
                >
                  <td
                    className="table-cell"
                  >
                    Fleurie Restaurant
                  </td>
                </TableCell>
                <TableCell
                  data="Charlottesville"
                  key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5-Charlottesville"
                >
                  <td
                    className="table-cell"
                  >
                    Charlottesville
                  </td>
                </TableCell>
                <TableCell
                  data="VA"
                  key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5-VA"
                >
                  <td
                    className="table-cell"
                  >
                    VA
                  </td>
                </TableCell>
                <TableCell
                  data="(434) 971-7800"
                  key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5-(434) 971-7800"
                >
                  <td
                    className="table-cell"
                  >
                    (434) 971-7800
                  </td>
                </TableCell>
                <TableCell
                  data="French,European,Cafe,Continental,American"
                  key="0f41a3d0-0641-4eef-b7fd-706f083cf0d5-French,European,Cafe,Continental,American"
                >
                  <td
                    className="table-cell"
                  >
                    French,European,Cafe,Continental,American
                  </td>
                </TableCell>
              </tr>
              <tr
                className="table-row"
                key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56"
              >
                <TableCell
                  data="The Capital Grille"
                  key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56-The Capital Grille"
                >
                  <td
                    className="table-cell"
                  >
                    The Capital Grille
                  </td>
                </TableCell>
                <TableCell
                  data="Dallas"
                  key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56-Dallas"
                >
                  <td
                    className="table-cell"
                  >
                    Dallas
                  </td>
                </TableCell>
                <TableCell
                  data="TX"
                  key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56-TX"
                >
                  <td
                    className="table-cell"
                  >
                    TX
                  </td>
                </TableCell>
                <TableCell
                  data="(214) 303-0500"
                  key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56-(214) 303-0500"
                >
                  <td
                    className="table-cell"
                  >
                    (214) 303-0500
                  </td>
                </TableCell>
                <TableCell
                  data="Steak,American"
                  key="0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56-Steak,American"
                >
                  <td
                    className="table-cell"
                  >
                    Steak,American
                  </td>
                </TableCell>
              </tr>
            </TableBody>
          </tbody>
        </table>
      </Table>
    </div>
  </div>
</Dashboard>
`;
